packages/react-components/src/components/MessageThread.tsx

```diff
@@ -860,14 +864,6 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
     }
   }, [numberOfChatMessagesToReload, onLoadPreviousChatMessages]);
 
-  const handleInfiniteScroll = useCallback((): void => {
```

Comment:
Hmm, totally forget why we added another function on top of fetchNewMessageWhenAtTop, looks like just a small check here, good to remove it